lib/client/states/state.py

```diff
@@ -11,7 +11,7 @@ def leave(self):
         #buh bye - cleanup stuff
         pass
 
-    def render(self):
+    def render(self, screen):
```

Comment:
That's probably true - I am just used to pygame frameworks where the context is globally available and renders are expected to handle their own business. I agree that in some cases it is better, was just wondering if it is needed.